org-formation/075-security-hub/security-hub-suppress-infra.yaml

```diff
@@ -484,14 +484,14 @@ Resources:
                 - prefix: 'arn:aws:s3:::mobiletoolbox.org'
                 - prefix: 'arn:aws:s3:::privacytoolkit.sagebase.org'
                 - prefix: 'arn:aws:s3:::privacytoolkit.sagebionetworks.org'
-                - prefix: 'arn:aws:s3:::prod*'
+              #  - prefix: 'arn:aws:s3:::prod*'
```

Comment:
This is matching on the prefix of the bucket name, so we should be able to remove just the `*` character